frontend/src/styles/UnreadMessagesIndicator.css

```diff
@@ -0,0 +1,16 @@
+.unread-messages-counter {
+    border: 1px solid blue;
+    color: blue;
+    width: 15px;
+    height: 15px;
+    border-radius: 50%;
+    text-align: center;
+    font-size: 10px;
+}
+
+.unread-messages-count {
+    vertical-align: 0%;
```

Comment:
I don't think we want to use `0%` here. Maybe some other value?
https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align